source/developer/slash-commands.rst

```diff
@@ -36,6 +36,7 @@ Each Mattermost installation comes with some built-in slash commands that are re
     "/logout", "Log out of Mattermost", "/logout"
     "/me *{message}*", "Do an action", "/me Hello World"
     "/msg *{@username}* *{message}*", "Send a Direct Message to a user", "/msg @alice hello"
+    "/groupmsg *{@username,...}* *{message}*", "Sends a Group Message to the specified users", "/groupmsg @alice,@bob hello"
```

Comment:
Not sure whether there should be a space between `@alice,@bob...`?

Reply:
Good catch. I have changed it. Probably is more clear that you can use spaces in this way, and with the "," is already clear that you can use it without spaces.